pkg/services/ngalert/api/api_testing_test.go

```diff
@@ -77,20 +176,19 @@ func TestRouteTestGrafanaRuleConfig(t *testing.T) {
 
 			evalFactory := eval_mocks.NewEvaluatorFactory(evaluator)
 
-			srv := createTestingApiSrv(ds, ac, evalFactory)
+			srv := createTestingApiSrv(t, ds, ac, evalFactory)
 
-			response := srv.RouteTestGrafanaRuleConfig(rc, definitions.TestRulePayload{
-				Expr: "",
-				GrafanaManagedCondition: &definitions.EvalAlertConditionCommand{
-					Condition: data1.RefID,
-					Data:      ApiAlertQueriesFromAlertQueries([]models.AlertQuery{data1, data2}),
-					Now:       currentTime,
-				},
+			rule := validRule()
+			rule.GrafanaManagedAlert.Data = ApiAlertQueriesFromAlertQueries([]models.AlertQuery{data1, data2})
+			response := srv.RouteTestGrafanaRuleConfig(rc, definitions.PostableExtendedRuleNodeExtended{
+				Rule:           rule,
+				NamespaceUID:   "test-folder",
+				NamespaceTitle: "test-folder",
 			})
 
 			require.Equal(t, http.StatusOK, response.Status())
 
-			evaluator.AssertCalled(t, "Evaluate", mock.Anything, currentTime)
+			evaluator.AssertCalled(t, "Evaluate", mock.Anything, mock.Anything)
```

Comment:
We can't know for sure the exact time, but it doesn't seem to be important (other tests also use `mock.Anything` in the same place).